### ISSUE TITLE
RP5 - Black Screen

### ISSUE THREAD

**Issue by Daholli** [worproject/rpi5-uefi#39]
[Step 4](https://github.com/worproject/rpi5-uefi#4-connect-peripherals-and-power-on-the-device)

Just does nothing for me, the monitor stays active, but I get no QR code, and no loading bar.
I double checked every step, when I have an installation medium the installation medium starts, but after I selected which kind of mode I want to run (in this case I am trying to install nixos-minimal).

I've spent a good time trying to fix it yesterday with no luck, in the meantime I booted up debian:
![image](https://github.com/user-attachments/assets/4bab9cc6-6e3a-4ea3-a9fd-89e2400fe620)

with the proprietary boot, so the pi does work, but changing the bootloarder does not seem to work

**Comment by zcop** [comment]
I had the same problem with Pi 5 2GB version, seam UEFI firmware didnt load > Black screen right after Pi Bootloader screen

**Comment by Daholli** [comment]
Have you managed to fix this? I have now switched to a non-uefi style setup, but I would still be interested in having the clean version

**Comment by zcop** [comment]
No, i found that Pi 5 with BCM2712 Stepping D0 that need "bcm2712d0-rpi-5-b.dtb" to boot, that make boot process go further but still not load RPI_EFI.fd

**Comment by Nadirisim** [comment]
i have the same issue with my pi 5 2gb has anyone found a solution yet?
